### ISSUE TITLE
cv2 issue in COLAB on "import deeplabcut"

### ISSUE THREAD

**Issue by byronrogers73** [DeepLabCut/DeepLabCut#1625]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Using the Colab on my own data. It was working fine (had trained to 400,000 iterations) until I restarted the runtime this morning. When it came to "import deeplabcut", it kicked up the error below. I tried uninstalling/reinstalling opencv.

### Operating System

MacOS

### DeepLabCut version

dlc version 

### DeepLabCut mode

single animal

### Device type

gpu in COLAB

### Relevant log output

```shell
The error on "import deeplabcut" below....

DLC loaded in light mode; you cannot use any GUI (labeling, relabeling and standalone GUI)
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-16-cfa4f159dfc5> in <module>()
----> 1 import deeplabcut

14 frames
/usr/local/lib/python3.7/dist-packages/cv2/__init__.py in <module>()
      7 
      8 from .cv2 import *
----> 9 from .cv2 import _registerMatType
     10 from . import mat_wrapper
     11 from . import gapi

ImportError: cannot import name '_registerMatType' from 'cv2.cv2' (/usr/local/lib/python3.7/dist-packages/cv2/cv2.cpython-37m-x86_64-linux-gnu.so)

---------------------------------------------------------------------------
NOTE: If your import is failing due to a missing package, you can
manually install dependencies using either !pip or !apt.

To view examples of installing some common dependencies, click the
"Open Examples" button below.
---------------------------------------------------------------------------


I tried removing and reinstalling cv2....the following output...and when I re-tried "import deeplabcut" the same error persisted.

Collecting opencv-python
  Downloading opencv_python-4.5.5.62-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (60.4 MB)
     |████████████████████████████████| 60.4 MB 1.2 MB/s 
Requirement already satisfied: numpy>=1.14.5 in /usr/local/lib/python3.7/dist-packages (from opencv-python) (1.19.5)
Installing collected packages: opencv-python
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
albumentations 0.1.12 requires imgaug<0.2.7,>=0.2.5, but you have imgaug 0.4.0 which is incompatible.
Successfully installed opencv-python-4.5.5.62
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by byronrogers73** [comment]
on pip install....i see the line...

Requirement already satisfied: opencv-python in /usr/local/lib/python3.7/dist-packages (from imgaug>=0.4.0->deeplabcut) (4.1.2.30)

But I see this error with Open CV where they state that _registerMatType is a part of OpenCV 4.5.4

https://github.com/opencv/opencv-python/issues/591

I uninstalled and successfully installed opencv-python-4.5.5.62

But the error still occurs.

I tried the following
import cv2
print(cv2.__version__)

and the same error kicks up on importing cv2

**Comment by scataldi88** [comment]
I have the same identical issue. I tried restarting runtime, factory reset, uninstalling packages and installing again..nothing worked.
Was anybody able to resolve?

**Comment by MMathisLab** [comment]
definitely a colab issue (imports as expected in terminal / worked in colab yesterday); will look into it.

**Comment by MMathisLab** [comment]
seems to be a multiple install issue (it might be a dependency now of DLC includes opencv as well)... 

in the meantime, this works:

(1) add this to the cell in colab where you install deeplabcut:

```python
!pip install deeplabcut
!pip uninstall opencv_python_headless
!pip install opencv-python-headless==4.5.4.60
```
(2) click, run and when prompted type "y", i.e.:
`Proceed (y/n)? y`

works:
<img width="1066" alt="Screen Shot 2021-12-29 at 12 16 25 PM" src="https://user-images.githubusercontent.com/28102185/147696183-88baee6b-327e-46af-bf45-30f69e5d72b6.png">


total:
<img width="1053" alt="Screen Shot 2021-12-29 at 12 17 54 PM" src="https://user-images.githubusercontent.com/28102185/147696208-847313df-2aab-4073-87d4-31ddf22be840.png">

**Comment by scataldi88** [comment]
This solution worked for me. Thank you

**Comment by MMathisLab** [comment]
formal fix in #1628 works as well on colab.
